src/pages/PatientVitals.js

```diff
@@ -141,34 +140,41 @@ const PatientVitals = ({
   return (
     <div className="wrapper">
       <div className="second-header">
-      {(progressedPage === NEW_PATIENT_PAGES.patientInfo || 
-       progressedPage === NEW_PATIENT_PAGES.covidHistory || 
-       progressedPage ===  NEW_PATIENT_PAGES.preExistingCondition || 
-       progressedPage === NEW_PATIENT_PAGES.allergy || 
-       progressedPage === NEW_PATIENT_PAGES.medication) && 
-        <div className="progress-bar">
-          <div className="progress" ></div>
-          <style>{`
+        {(progressedPage === NEW_PATIENT_PAGES.patientInfo ||
```

Comment:
these indentation updates can cause merge conflict with another PR, @Shreets can you please revert them?